Balancer-Bots/Swap-FlashLoan-Fee-Change/src/agent.ts

```diff
@@ -0,0 +1,53 @@
+import { BlockEvent, ethers, Finding, getEthersProvider, HandleBlock, Initialize } from "forta-agent";
+import CONFIG from "./agent.config";
+import { FLASH_LOAN_FEE_PERCENTAGE_CHANGED_ABI, SWAP_FEE_PERCENTAGE_CHANGED_ABI } from "./constants";
+import { NetworkManager } from "forta-agent-tools";
+import { createFinding, NetworkData } from "./utils";
+
+const networkManager = new NetworkManager<NetworkData>(CONFIG);
+
+export const provideInitialize = (
+  networkManager: NetworkManager<NetworkData>,
+  provider: ethers.providers.Provider
+): Initialize => {
+  return async () => {
+    await networkManager.init(provider);
+  };
+};
+
+export const provideHandleBlock = (
+  provider: ethers.providers.Provider,
+  networkManager: NetworkManager<NetworkData>
+): HandleBlock => {
+  const protocolFeesCollectorIface = new ethers.utils.Interface([
+    SWAP_FEE_PERCENTAGE_CHANGED_ABI,
+    FLASH_LOAN_FEE_PERCENTAGE_CHANGED_ABI,
+  ]);
+
+  const sighashes = [
+    protocolFeesCollectorIface.getEventTopic("SwapFeePercentageChanged"),
+    protocolFeesCollectorIface.getEventTopic("FlashLoanFeePercentageChanged"),
+  ];
+
+  return async (blockEvent: BlockEvent): Promise<Finding[]> => {
+    const logs = (
+      await provider.getLogs({
+        address: networkManager.get("protocolFeesCollectorAddress"),
+        fromBlock: blockEvent.blockNumber,
+        toBlock: blockEvent.blockNumber,
+      })
+    ).filter((log) => sighashes.includes(log.topics[0]));
```

Comment:
please include the topics in the filter:
```suggestion
    const logs = (
      await provider.getLogs({
        address: networkManager.get("protocolFeesCollectorAddress"),
        fromBlock: blockEvent.blockNumber,
        toBlock: blockEvent.blockNumber,
        topics: [sighashes],
      })
    );
```

Reply:
Good catch, I forgot to change this in this one too when I saw there could be multiple options for a topic. Thanks!